devops/src/ax/devops/gateway/rest.py

```diff
@@ -0,0 +1,470 @@
+import boto3
```

Comment:
I'm assuming the contents of this file is simply moving existing code into the corresponding flask endpoints? Any logic changes?

Reply:
For this pull request, no. I want to make sure what was working is still working.